pkg/plugin/plugin.go

```diff
@@ -135,6 +135,9 @@ func (p *plugin) CreateOrUpdate(ctx context.Context, cs *api.OpenShiftManagedClu
 			return err
 		}
 	}
+	if cs != nil {
```

Comment:
Remove?

Reply:
+1.  The setting of cs.Properties.NetworkProfile.VnetID will happen at the bottom of this function.

Reply:
removed Debug output